db/seeds.rb

```diff
@@ -90,13 +90,15 @@ def create_loan_requests_for_each_borrower(quantity)
     end
   end
 
-  def create_orders(quantity)
-    quantity.times do
-      borrowers.each do |borrower|
+  def create_orders
+    loan_requests = LoanRequest.all
+    8.times do
+      loan_requests.each do |request|
+        donate = (3..25).to_a.sample.to_s
         lender = User.where(role: 0).order("RANDOM()").take(1).first
-        order = Order.create(cart_items: { "#{borrower.id}" => "25" }, user_id: lender.id)
+        order = Order.create(cart_items: { "#{request.id}" => donate }, user_id: lender.id)
         order.update_contributed(lender)
-        puts "Created Order for Borrower #{borrower.name} by Lender #{lender.name}"
+        puts "Created Order for Request #{request.title} by Lender #{lender.name}"
```

Comment:
Line is too long. [82/80]<br>Do not write to stdout. Use Rails' logger if you want to log.